kerby-kerb/kerb-client/src/main/java/org/apache/kerby/kerberos/kerb/client/KrbClientBase.java

```diff
@@ -271,15 +271,22 @@ public void storeTicket(TgtTicket tgtTicket,
      */
     public void storeTicket(SgtTicket sgtTicket, File ccacheFile) throws KrbException {
         LOG.info("Storing the sgt to the credential cache file.");
-        if (!ccacheFile.exists()) {
+        boolean isFreshNew = !ccacheFile.exists() || (ccacheFile.length() == 0);
```

Comment:
Please change "isFreshNew" to "createCache".

Reply:
done